pkgs/development/typst-packages/cetz.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildTypstPackage,
+  fetchFromGitHub,
+  just,
+  file,
+  perl,
+}:
+
+buildTypstPackage rec {
+  pname = "cetz";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cetz-package";
+    repo = "cetz";
+    rev = "v${version}";
+    hash = "sha256-5+Np64+0Ca8yL7D1CCUewF03Wh6BM2bV1BQNUA+bW74=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    just
+    file
+    perl
+  ];
+
+  prePatch = ''
+    patchShebangs scripts/package
```

```suggestion
    patchShebangs --build scripts/package
```
Also ensure that all interpreters have been added to `nativeBuildInputs`.